src/Runtime/Runtime/System.Windows/TextMeasurementService.cs

```diff
@@ -265,7 +265,10 @@ public bool IsTextMeasureDivID(string id)
             else
                 savedTextBlockPadding = padding;
 
-            string javaScriptCodeToExecute = $@"document.measureTextBlock(""{strText}"",""{strFontSize}"",""{strFontFamily}"",""{strFontStyle}"",""{strFontWeight}"",""{strTextWrapping}"",""{strPadding}"",""{strWidth}"",""{strMaxWidth}"")";
+            // UGiacobbi 18022022 If we pass a quoted font family we need to escape it
+            var escapedstrFontFamily = strFontFamily.Replace("\"", "\\\"");
```

Comment:
Escapes potential other forbidden characters
```suggestion
            var escapedstrFontFamily = INTERNAL_HtmlDomManager.EscapeStringForUseInJavaScript(strFontFamily);
```